### PR TITLE
add clean theme

### PR DESCRIPTION
**Pull request by knaman2609**

### CODE REVIEW

**Comment by geraldb** [comment]
@knaman2609  While you're waiting to get accepted on jekyllthemes.org - I invite you to send a pull request to the alternative [Dr. Jekyll's Themes](http://drjekyllthemes.github.io) directory. Same system. Just a little simpler, easier. Your Clean theme is welcome. Cheers.

File: _posts/2016-09-27-clean.markdown [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title: Clean
+date: 2016-09-27
```
Comment (gynter) [inline _posts/2016-09-27-clean.markdown]: Thanks for PR! For the sake of common formatting, would You please be so kind and add a timestamp to this date? If You don't know the exact time then please set it as 00:00:00.

Thanks!

**Comment by gynter** [comment]
The thumbnail image dimensions should be 250px × 200px not 250px × 250px.

**Comment by knaman2609** [comment]
@gynter  thumbnail size and post time fixed.

File: _posts/2016-09-27-clean.markdown [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+layout: post
+title: Clean
+date: 2016-09-27 00:00:00
+homepage: https://github.com/knaman2609/clean  
+download: https://github.com/knaman2609/clean/archive/master.zip
+demo: http://knaman2609.github.io/clean/
+author: knaman2609
+thumbnail: clean.png
+license: MIT License
+---
```
Comment (gynter) [inline _posts/2016-09-27-clean.markdown]: Hello! If You don't have a license file in Your repository, then please add http://opensource.org/licenses/MIT as `license_link`. Thanks!

**Comment by knaman2609** [comment]
@gynter licence link pushed.

**Comment by gynter** [comment]
Thanks! @mattvh :shipit:

**Pull request merged**